tests/unit/test_helpers.py

```diff
@@ -159,7 +159,7 @@ def test_get_browser_name(self):
         self.assertEqual(get_browser_name(FIREFOX_MOBILE_UA_OLD),
                          'firefox mobile')
         self.assertEqual(get_browser_name(FIREFOX_TABLET_UA),
-                         'firefox mobile (tablet)')
```

Comment:
We should probably figure out why this changed, and if we care.

Reply:
```python
FIREFOX_TABLET_UA = 'Mozilla/5.0 (Android 4.4; Tablet; rv:41.0) Gecko/41.0 Firefox/41.0'  # noqa
```

which is definitely a tablet. I'm not sure why we get `firefox mobile (tablet)` instead of just `firefox tablet` but that's probably a separate issue. 

https://github.com/webcompat/webcompat.com/blob/94911bcce14261dcc8a18a61fc9fcc725716dfd2/webcompat/helpers.py#L162-L172

ok 

```python
        return get_browser(user_agent_string).rsplit(' ', 1)[0].lower()
```

Let's go see what `get_browser` returns.
https://github.com/webcompat/webcompat.com/blob/94911bcce14261dcc8a18a61fc9fcc725716dfd2/webcompat/helpers.py#L139-L159

* UA parser: 0.8
```python 
{
    'user_agent': {
        'family': 'Firefox Mobile', 
        'major': '41', 
        'minor': '0', 
        'patch': None
    }, 
    'os': {
        'family': 'Android', 
        'major': '4', 
        'minor': '4', 
        'patch': None, 
        'patch_minor': None
        }, 
    'device': {
        'family': 'Generic Tablet', 
        'brand': 'Generic', 
        'model': 'Tablet'
        }, 
    'string': 'Mozilla/5.0 (Android 4.4; Tablet; rv:41.0) Gecko/41.0 Firefox/41.0'
    }
```

* UA parser: 0.9

```python
{
    'user_agent': {
        'family': 'Firefox Mobile', 
        'major': '41', 
        'minor': '0', 
        'patch': None
        }, 
    'os': {
        'family': 'Android', 
        'major': '4', 
        'minor': '4', 
        'patch': None, 
        'patch_minor': None
        }, 
    'device': {
        'family': 'rv:41.0', 
        'brand': 'Generic_Android', 
        'model': 'rv:41.0'
        }, 
    'string': 'Mozilla/5.0 (Android 4.4; Tablet; rv:41.0) Gecko/41.0 Firefox/41.0'
    }
```


Yoohoo… that device is wow… Probably a bug in `user_agent_parser.Parse(user_agent_string)`

We could freeze to `0.8` then file a bug on the project. and when they fix it we can upgrade.

Reply:
https://github.com/ua-parser/uap-python/issues, I'll check today if i can do a pull request on ua_parser

Reply:
Opened https://github.com/ua-parser/uap-python/issues/84

Reply:
opened 
Issue: https://github.com/ua-parser/uap-core/issues/461
PR: https://github.com/ua-parser/uap-core/pull/462